packages/custodyKeyring/src/interfaces/ICustodianEnvironment.ts

```diff
@@ -0,0 +1,17 @@
+export interface ICustodianEnvironment {
+  type: string;
+  name: string;
+  onboardingUrl: string;
+  website: string;
+  envName: string;
+  apiUrl: string;
+  apiVersion: string;
+  iconUrl: string;
+  displayName: string;
+  production: boolean;
+  refreshTokenUrl: string;
+  websocketApiUrl: string;
+  isNoteToTraderSupported: boolean;
+  custodianPublishesTransaction: boolean;
+  version: number;
+}
```

Comment:
This type represents a flattened custodian environment. It's been added to `MMIConfiguration` setup and can be assumed that `state.mmiConfiguration.custodians` is of this type.